samples/drivers/espi/sample.yaml

```diff
@@ -1,8 +1,13 @@
 sample:
+  description: A basic demo to showcase eSPI driver usage and handshake with
+    eSPI master
   name: eSPI driver sample
 tests:
   sample.drivers.espi:
-    tags: drivers
+    tags: drivers espi
+    filter: dt_compat_enabled_with_alias("gpio-espi", "gpioespi0") and
```

Comment:
filter here should just be something like `dt_compat_enabled("microchip,mec15xx-board-power")`